spec/support/helpers/feature_helpers.rb

```diff
@@ -5,8 +5,11 @@ def screenshot_after
     yield
 
     if @metadata_screenshot
+      @screenshot_index = defined?(@screenshot_index) ? @screenshot_index + 1 : 1
       example_text, spec_path = inspect.match(/"(.*)" \(\.\/spec\/features\/(.*)_spec\.rb/)[1, 2]
-      page.percy_snapshot("#{spec_path}/#{example_text.parameterize}#{current_path}")
+
+      screenshot_name = "#{format('%02d', @screenshot_index)}|#{spec_path}|#{example_text.parameterize}|#{current_path.parameterize}"
```

Comment:
Changed how the naming is laid out to be more readable, for example:

```
37|web_intake/new_joint_filers|new-client-filing-joint-taxes-with-spouse-and-dependents|en-dependents

```

Reply:
I would rather a different character to split these, not `|` which is used by bash, so it would require more special encoding.

I would suggest:

```
#{spec_path.parameterize}/#{example_text.parameterize}/#{format('%02d', @screenshot_index)}-#{current_path.parameterize}`

It seems to me there's a natural grouping - spec path first, then example name, then screenshot index (with current path being an optional suffix to screenshot index).

I think that it's best to avoid unnecessary `/` characters and to avoid `|`, but I don't know if these end up as filenames on disk, or what.

100% not a blocker.